Lib/threading.py

```diff
@@ -1346,6 +1347,26 @@ def enumerate():
     with _active_limbo_lock:
         return list(_active.values()) + list(_limbo.values())
 
+
+_threading_atexits = None
```

Comment:
I don't think there is any downside to assigning an empty list directly here?

Reply:
I did this so that a list is not spawned every time `threading` is imported, even when `_register_atexit()` is not used. It's not very substantial, but I've always made the habit of avoiding as much overhead as possible in module-level globals. IMO, it's one of those practices that are individually insignificant, but can add up over time.

That being said, I would be okay with just initializing the list directly here, since the overhead for creating a single list is so small. I'll leave it up to you, as the primary maintainer for `threading`.

Reply:
I think initializing the list here is simpler. The overhead of a single (empty) list object is tiny.